release-notifier/internal/psql/psqlnotifier.go

```diff
@@ -48,3 +55,39 @@ func GetLatestPSQLRelease() string {
 	c.Visit(website)
 	return release
 }
+
+func GetPrevRelFromArtifact(artifact string) string {
```

Comment:
Why is the file name passed to `GetPrevRelFromArtifact`, but the Save counterpart "knows" the filename. 
I would just store the filename in a constant and use it in both functions instead of passing it to one